common/reactive/src/main/java/io/helidon/common/reactive/CompletionAwaitable.java

```diff
@@ -294,6 +294,18 @@ void setOriginalStage(final Supplier<CompletionStage<T>> originalStage) {
         return new CompletionAwaitable<T>(() -> completionStage, this);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public CompletionAwaitable<T> exceptionallyAccept(final Consumer<Throwable> consumer) {
```

Comment:
Is this missing a `@Override` annotation ?

Reply:
No this one is ours, but the inherit docs was wrong thx